Quick-Start/Anatomy_of_extension.md

```diff
@@ -1,418 +1,418 @@
+ 
+## Creating the SQL-Script File for PostgreSQL Extension: 
+Writing a SQL script file is an essential part of creating a PostgreSQL extension. It helps developers define the objects and behaviour of the extension by using SQL statements. Here's a quick start guide on how to write a SQL script file for creating a PostgreSQL extension: 
+ 
+1.  **Define the extension's requirements:** Before writing the SQL script file, make sure you have a clear understanding of the requirements and functionality of your extension. Determine what objects, such as tables, functions, or data types, need to be created or modified. 
+ 
+2.  **Choose a file name and extension:** Select a meaningful name for your SQL script file, typically with a `.sql` extension. This will help organize and identify the file easily. 
+ 
+3.  **Set up the necessary headers:** Begin the SQL script file with a comment block that includes information about the extension, such as its name, version, and purpose. This header will provide important details about the script file to other developers or users. 
+ 
+4.  **Write SQL statements:** In the script file, write SQL statements to define the desired objects and behaviour of the extension. For example, you can use `CREATE FUNCTION` statements to define custom functions, `CREATE TABLE` statements to create tables, or `CREATE TYPE` statements to define custom data types. 
+ 
+5.  **Organize the script file:** It's recommended to organize the SQL statements logically within the script file. You can group related statements together or separate them based on their purpose. Using comments to provide explanations or section headers can make the script more readable.  
+ 
+6.  **Add necessary comments:** Include comments within the script file to provide explanations or clarify the purpose of specific SQL statements. This helps other developers understand the code and makes it easier to maintain and troubleshoot the extension. 
+ 
+7.  **Test the SQL script:** Before finalizing the SQL script file, it's essential to test it to ensure that the SQL statements execute successfully without any errors. Use a tool like the `psql` command-line client or a PostgreSQL GUI to execute the script and verify its functionality. 
+ 
+By following this quick start guide, you can create a well-structured and functional SQL script file for your PostgreSQL extension. It helps developers define the extension's behaviour, create necessary objects, and implement the desired functionality within the PostgreSQL database. 
+ 
+# 2) Control File: 
+ 
+A Control file, also known as a control file template or control file script, is an essential component of a PostgreSQL extension. It provides metadata and configuration information necessary for the installation, upgrade, and management of the extension. 
```

Comment:
I don't think you need to list the other names the control file is known as. I've not heard of any of these myself.